src/time/span.cr

```diff
@@ -286,6 +286,10 @@ struct Time::Span
   def self.zero
     new(0)
   end
+              
+  def zero?
+    self == typeof(self).zero
```

Comment:
@Sija `self == typeof(self).zero` is a general implementation because it works on all types with a self.zero implementation. I was saying that there's no point in using a more general implementation like this versus `@ticks == 0`, unless you move `#zero?` to `Object`.